### ISSUE TITLE
Incompatibility with Either Galleries or a Photo Page

### ISSUE THREAD

**Issue by ymolodtsov** [jsonbecker/plugin-glightbox#3]
So now we have two options for placing an image: either a Markdown image or a Hugo shortcode. Both seem to have certain limitations and right now I'm not seeing a possible way to rectify this.

If you use a Hugo shortcode, then these photos aren't picked up on the Photos page, because Hugo doesn't recognize them.

If you use a Markdown insertion, then the photo will be shown on the Photos page. But because of the way Markdown syntax processing works, you can't have a div with an img-gallery class right before. Markdown expects an empty line between the previous text and the images and creates an empty <p> there that ruins the layout. 

I'd be happy to contribute a solution but at the moment I'm not even sure if there's a clean way.

**Comment by jsonbecker** [comment]
Yup-- I don't know the solution right now. If I had one, I'd be putting it into place. Happy for collaborators or ideas.

**Comment by jsonbecker** [comment]
> So now we have two options for placing an image: either a Markdown image or a Hugo shortcode. Both seem to have certain limitations and right now I'm not seeing a possible way to rectify this.
> 
> If you use a Hugo shortcode, then these photos aren't picked up on the Photos page, because Hugo doesn't recognize them.
> 
> If you use a Markdown insertion, then the photo will be shown on the Photos page. But because of the way Markdown syntax processing works, you can't have a div with an img-gallery class right before. Markdown expects an empty line between the previous text and the images and creates an empty
> 
> there that ruins the layout.
> 
> I'd be happy to contribute a solution but at the moment I'm not even sure if there's a clean way.

If you use a div and enter new lines between everything, the galleries work. For example:

```html
<div class="img-gallery">

![](urltoimg1.jpg)

![](urltoimg2.jpg)

</div>

```

Will render properly like the shortcode would have.
